modules/vstudio/vs2010_vcxproj.lua

```diff
@@ -133,13 +133,14 @@
 			m.preferredToolArchitecture,
 			m.latestTargetPlatformVersion,
 			m.windowsTargetPlatformVersion,
+			m.fastUpToDateCheck,
 		}
 	end
 
 	m.elements.globalsCondition = function(prj, cfg)
 		return {
 			m.windowsTargetPlatformVersion,
-			m.xpDeprecationWarning,
+            m.xpDeprecationWarning,
```

Comment:
This should be reverted, this change is wrong and unnecessary.

Reply:
Good catch, thought I had reverted this.